src/stacks-hierarchy/Root_LoginActiveFareContractWarningScreen.tsx

```diff
@@ -126,27 +118,24 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
     marginVertical: theme.spacings.medium,
     textAlign: 'center',
   },
-  errorMessage: {
-    marginBottom: theme.spacings.medium,
+  overlay: {
```

Comment:
Could we make the ticket transparent instead of adding a overlay?

Reply:
I tried adding `transparent` to the `<GenericSectionItem>` in `<FareContractView>`, also tried to set the opacity there but that didn't work / look so good 😢 @gorandalum suggested the overlay approach, but if you can give me some pointers on other approach with transparent I can try that 😄

Reply:
You can add opacity to the `fareContract` style

Reply:
Done 💯

Reply:
Not totally sure about using opacity instead of overlay, at least need to check it. The reason it works pretty ok for AtB is that `background_accent` is actually a dark color, but how does it work for Fram?

<img width="278" alt="Screenshot 2024-03-18 at 09 11 21" src="https://github.com/AtB-AS/mittatb-app/assets/675421/014505cd-8be3-4f7c-ae81-6d8cd6f06c9e">

Reply:
Talked to @hannahATB and we figured this works fine 😄 FRAM normal vs dark mode👇🏼 
<img src="https://github.com/AtB-AS/mittatb-app/assets/126664682/2c8b091b-7fee-4d70-99fc-70d6ba5dcbb4" width="200"/> <img src="https://github.com/AtB-AS/mittatb-app/assets/126664682/899835d5-2a27-408d-924e-b22c1da3d104" width="200"/>